user-registration-app/src/app/views/signup-view/signup-view.component.ts

```diff
@@ -73,4 +75,10 @@ export class SignupViewComponent extends BaseComponent implements OnInit {
       this.progressBar.complete();
     }
   }
+  onEmailEntered(email: string) {
+    if (/@psu.edu$/.test(email)) {
```

Comment:
Don't think so. This is also already in /login so I'm assuming it works. I also just googled it and it doesnt seem like it does

Reply:
I'm confused as to what the problem is. When I "jjj@psu.edu" I get a match and when I try a different email "jjj@email.com" I don't get a match which is what is supposed to happen

Reply:
It's minor but basically it'll also match @psumedu or @psu(insert symbol)edu

Reply:
ohhhh I see. Yeah that's minor enough that it can go into a different PR I dont think I'm willing to PR for the fix, and I'm assuming most users will have x.y and then go to password, which is when the flag pops up. So if they're in the middle of typing the popup wont appear until they go into the password field anyway